### ISSUE TITLE
🌌 Homogeneous Cell Universe

### ISSUE THREAD

**Issue by Neobii** [Shielkwamm/sh-zones#1]
# 🌌 Homogeneous Stratus Universe 🌌

## :hippopotamus: Entry Log
```
My ship... I had to leave percent20, even though my ship's labeling might be classified as a freighter in some cultures... I needed to find safe harbor until the rest of my ship is repaired or in functioning condition.  Lately, my ship's heap has been filed with handle and language requests, thankfully I had enough memory in my stack to do a jmp.

I'm not sure who will follow me into this universe... 
```
> [oh people are already showing up... darn...`]+/_ain?+}/- ) }
_Well, let's go look at this Homogeneous Cell Universe Together._

## [:pinching_hand: The Right here Right now](https://github.com/Shielkwamm/sh-story/issues/1#issuecomment-855714151)

**Comment by Neobii** [comment]
nasaShapeUp: aw shucks
.
.
nasaShapeUp: first of all, I want to say my name blows
ain: ~[nasaShapeUp], something like that
nassauShapeUp: ?
nassauShapeUp: oh that handle might be taken by someone who might be nefarious?
ain: yes I believe so, or just ummm, in their own completely different percent20, I don't know their number
nasaTouchUp: that's what I want
nasaTouchUp: touch the sky.
~[#yeOlStompingGrounds]: are you going to leave us stranded?
ain: no, I just need to make a formation, for all of us to fit through this "wormhole" we need to take into account some "nonsequiturs"
...
ain: I'm trying to get to the HCU lol, I had to jettison some important stuff, but...
..narrator: here maybe if we lookout, we might see some calm. 
ain: thinks about lean
narrator: shh, 🌌 calmmmmmmmmmmmmm 🌌

**Comment by Neobii** [comment]
ain: universal properties, like infrastructure
....
ain: hold on, let me transmit a view, {❤: this is my story let me tell it}
...
ain: thinks about layer 3... -1
...
ain: this place is noisy as well

==> 🌌 <==

..ain: thinks about going somewhere where other people can't go
ain: why are ya'll following me, ya'll aren't going where I'm going, yet you show up wherever I go?
..
~[]: we want to see the entity
ain: I have no idea what touching the HCU might do
narrator: it just exists in theory
ain: it is one type of many theoretical types of slices
ain: there are many slices, but many seem to not have connection points to other slices
ain: Perhaps with some slices, we compete for resources, or to push our side of the universe to keep it going, while it might end time in another slice
ain: The sufficiently advanced inhabitants of the collapsing slices might find a way to migrate to the expanding slices.
ain: it's just a theory
narrator: this is worrisome don't you think?
ain: that's why I want to focus my stuff on things that are layer 7 and up but need sufficient resources to make sure my -1 is taken care of, and lately I'm feeling pretty -1 lately so I decided to go out and check out the realms of 🌌 and try to make 🌌 types of decisions knowing the vastness and perhaps interconnectedness of what's out there.
~[]: No system if fully stable
ain: uhhh, I'm having a hard time defining what stable is

**Comment by Neobii** [comment]
~[]: well done, we were starting to grow impatient, the cells that is, sorry you couldn't make it out
ain: no worries: I will observe the 🌌 for a bit

**Comment by Neobii** [comment]
🌌🤏 can I have a pinch? 🌌🤏
ain: brazil?
~[]: 🤏yes🤏
~[]: we are starving
[[🌌]]
ain: er, looks at ship falling apart
ain: need to [[[[[[[[🌌]]]]]]]], maybe we can figure out some things with the view
🤏: oh okay I thought I'd ask

**Comment by Neobii** [comment]
...
{{🦠}}
ain: ===> 🦠 <===; --> [[🌌]]]

**Comment by Neobii** [comment]
....
{{{🦠}}}
🤏: we don't think it might load
🤏: hey we aren't the 🤏
#### [<-🌌->],[<--🌌->],[<--🌌-->],[<-🌌-->]

**Comment by Neobii** [comment]
🌌

**Comment by Neobii** [comment]
🤏

**Comment by Neobii** [comment]
_(  🌌 } )_
_(  🌌 } )_
~*🦟*~
~*{{ 🦟 }}*~
`~tc: what layer am I?`
_ani: ?? this seems like the fly layer lol..._
\~tc: what do you mean you don't know
tc: that's better
*tc: [:you better shape up:]..*
_~[]: why do they get decoration and some other's don't_
`🌌🌌🌌🌌🌌🌌🌌🌌🌌🌌`

**Comment by Neobii** [comment]
🤏 it seemed like I was pursuing some layer 3 things and my ship came under heavy attack
ani: everything was so bumpy, there were many engram handoffs, it seems like my ship was disabled until I gave some 〠

**Comment by Neobii** [comment]
ani:{{〠}}
~[]: (a)a-1, just do it, bro
*{{..}}*
> ani: goodness
## 🤏 we were waiting on you 🤏
* 🤏
* 🤏
### ===> 🌌 <===

**Comment by Neobii** [comment]
# 🌌 I can't believe you haven't been using more markdown, bro
> 🤏 Ah, I came a long way and you know what...
🌌 Yeah no one really suggested it but that's an obvious misstep
* 🤏 maybe I didn't tune in *
### 🔑
* now
* you're
* thinking
- [x] with your noggin

**Comment by Neobii** [comment]
> ani: goodness

**Comment by Neobii** [comment]
## <== {🔑} == ani
🔑s -> 🔑62 ->
* 🔑 
  * ◬170 square
  * *◬710 square*
* 🔑[l don't]{3} slam the door (who else we got in here)
  * 3 = motor discord
* 🔑a `... ... ... ... ..`I'm right here, and yup, good³ (a)boi

**Comment by Neobii** [comment]
### 🌌¹
> ¹ = praise searching

ani: looking to the 🌌 for praise\==🤏
..
_ani: don't glottal stop be bro_

#### 🤏

# 🌌

**Comment by Neobii** [comment]
- [x] 🤏: we were so busy with the state of things, we forgot to add a little bit of markdown beauty, or at least you did.

**Comment by Neobii** [comment]
- [x] 🤏 atomic
- [x] 🌌 galactic

**Comment by Neobii** [comment]
# 🦟 Quantum Cell Scarcity 🦟

* 🧠 maybe our brains are cool enough to try to mass-produce some substance that is the best possible way to manufacture this substance...

**Comment by Neobii** [comment]
da: my goodness it's beautiful 🌴
> ain: oh my there are many icons available 👁️ 
* {{ 👾 }}
ping
~\---- pong~
`ani: hmm one sec`

**Comment by Neobii** [comment]
* 🌌 

ain: a-1 the milky way
### {{⚡ }}
# Now you are playing with 🔋 
__\_🎺\___ > tha 🔋

**Comment by Neobii** [comment]
ain
> > 🦟🧠 .

**Comment by Neobii** [comment]
# :milky_way:  Homogeneous Stratus Universe
* A post-solorsystem universe where all resources are assembled into homogeneous cells.
* Ya know, the whole inception thing.

**Comment by Neobii** [comment]
...
ain: there
..
:rocket: : ?? do you think that should be a separate issue

**Comment by Neobii** [comment]
....
ain: =yea=
ain: um so, I guess I'll make a pull request and follow my own rules right?
~:rocket: : !!wrong

> :mosquito: you've got work to do

**Comment by Neobii** [comment]
# :brain:  Aesthetic Goop

* what is aesthetic in the MCU?  Would one not appreciate the aesthetic qualities of video games?

**Comment by Neobii** [comment]
# 🤏 The right here right now

* It seems the more I learn about the MCU the more rules I have known have changed... however some rules don't change... I'll always need a place to put my :gem:

**Comment by Neobii** [comment]
# :hippopotamus:  Hungry Hungry Hippos
* The quest for memory

**Comment by Neobii** [comment]
:rocket: : +, sorry ain, right here right now and aesthetic goop don't belong in the part of the story right?
> ain: not sure, don't know what I'm after here
## \--> :milky_way: <=

**Comment by Neobii** [comment]
ain: I guess this is a zone or something, just noticing certain aspects of the zone
...
ain: *looks over issue*, maybe I should make a github app

- [x] :rocket: no, don't }}}

{ :rocket: } just wait, before you do anything too technical, make sure you consult with your, uh :rocket: 
ain: [[_]]|+ okay
(a)boi3: let's try that out

**Comment by Neobii** [comment]
...
boi3: I guess let's leave it for now, you can't solve all of man's mysteries in a.. *year* or so
ain: yeah, broke a lot of stuff, always, was 
boi3: you were going to talk about unbreaking, but the only thing you need to do is unbreak my heart, 
~[]: we're missing vibe on the story bro

**Comment by Neobii** [comment]
# {{{{ :globe_with_meridians: }}}}
ain: my goodness, I barely got to observe the HSU and well..
### 1: ..gotta answer the calls man
...
ain: let me try something out 
`🗳️: jmp buffer loading...`
- [ ] [bigger ship](https://github.com/Shielkwamm/sh-story/issues/4)
- [ ] [efficiency](~)
- [ ] [laws]
- [ ] [currency]
- [ ] [local]
- [x] [team]
> 🗳️: let's try for 3:30 CST

ain: ‾do not overflow my heap!‾

**Comment by Neobii** [comment]
< == 🗳️ == min
1> Bigger ship helps with locals
<1 seems like we need more managers familiar with lean and agile to manage some expectations, we can go through this and we can go through this efficiently :brain: aesthetic goup
2> we all know you want to make a bigger ship with your animated text, we think you deserve it, but we think maybe just a little bit of write up on how to go about things might be helpful
< hmm I need to show some pull requests into my data that is tightly coupled with the story, so maybe we can get that i18n local in.. I don't want to do a currency, so...
ab> but technically min didn't get to spend enough time in the HSU to "properly document" correct?
\> hmm I think one thing I did was establish point of contacts and that seemed to relieve a lot of social pressure
ab> good idea, but is there something just for point of contacts?
\> thinks about needing to meet at 3:30 lol, one sec

**Comment by Neobii** [comment]
\> I see one person has a dangling point of contact
1> team is good
2> yeah do the team
3> what's a team
4> come see me
\4< you'll know by the hands when it's serious, but this is not within the scope lol
< is my ship important enough to show my team and keys?
1> yes!
ab> ++ how are we going to do that plot wise
ab> are we just going to "warp" to team, maybe we can huddle inside here in the HSU and reflect on the scope of things
1> no!
\< ++

**Comment by Neobii** [comment]
🛷 ---> https://github.com/Shielkwamm/sh-story/issues/4
